docs/integrations/destinations/clickhouse-migrations.md

```diff
@@ -1,54 +1,88 @@
-naming semantics, we will not remove existing data. If you would like to, you
-will need to delete all the tables saved in the old format, which for most
-people should be under `airbyte_internal.{database}_raw__`, but may vary based
-on your specific configuration.
+While this is a breaking change, existing connections continue to function after upgrading. However, data is written to a completely different location in a different format. **You must update any downstream pipelines** (SQL queries, BI dashboards, data transformations) to reference the new table locations and schema structure.
 
-## Gotchas
+### Migrating Existing Data to the New Format
 
-### Namespaces and the default database
+Airbyte cannot automatically migrate data from the v1 raw table format to the v2 typed table format. To get your data into the new format, you must perform a full refresh sync from your source.
 
-In V2 namespaces are treated as equivalent to a ClickHouse "database". This
-means if you set a custom namespace for your connection that will be the
-database the connector will use for queries instead of the "database"
-configured in the Destination settings.
+**Migration steps:**
 
-Previously, namespaces where added as a prefix to the table name. If you have
-existing connections configured in this fashion you may want to remove them.
+1. Upgrade your ClickHouse destination connector to version 2.0.0 or later
+2. Trigger a full refresh sync to populate the new typed tables
+3. Verify the new tables contain the expected data
+4. Update your downstream pipelines to reference the new table locations
+5. Optional: Remove the old v1 raw tables (see below)
 
-### Hostname
+#### Optional: Removing the Old v1 Raw Tables
 
-If the "Hostname" property in your configuration contains the protocol ("http
-or "https"), you will need to remove it.
+The v2 connector does not automatically remove tables created by the v1 connector. After successfully migrating to v2 and verifying your data, you can optionally remove the old raw tables created by v1 to free up storage space.
 
-The previous versions incidentally tolerated the protocol being stored in the
-hostname field.
\ No newline at end of file
+For most users, old tables are located in the `airbyte_internal` database with names matching the pattern `airbyte_internal.{database}_raw__stream_{table}`. However, the exact location may vary based on your v1 configuration.
+
+:::caution
+Always verify that you have successfully migrated your data before removing old tables. Once dropped, the data cannot be recovered without re-syncing from your source.
+:::
+
+To remove old tables:
+
+```sql
+-- List tables in the airbyte_internal database
+SHOW TABLES FROM airbyte_internal;
+
+-- Drop individual tables
+DROP TABLE airbyte_internal.{database}_raw__stream_{table};
+
+-- Or drop the entire airbyte_internal database if it only contains old Airbyte tables
+DROP DATABASE airbyte_internal;
```

Comment:
Users should verify no syncs are running before they do this. The current connector does still use the `airbyte_internal` database for temporary scratch space (dedup streams will stage data into this database, and truncate refreshes / overwrite syncs will store data here as well).